conf/modules.config

```diff
@@ -18,6 +18,7 @@ re114
     Replace ASAP with the new output publishing!
 */
 mf {
```

Comment:
Function name `mf` doesn't tell me what it's doing, even if it's being replaced soon, maybe something like `dir_builder` or `path_builder`.

Reply:
Honestly couldn't tell you that I remember what it means either... (meta formatter?) I chose it because it was short though so I could have better line lengths 😂 

I'll leave it as it for now, as it's not even a supported Nextflow feature - it's just a hack Mahesh and I came up with to add functions to the config file to avoid re-writing boilerplate. It needs to go ASAP as it's not supported by strict syntax.

Reply:
No problem, I'll keep an eye out for it.